src/cltkv1/morphology/Morphosyntax.py

```diff
@@ -0,0 +1,330 @@
+"""A module for representing universal morphosyntactic feature bundles."""
+
+__author__ = ["John Stewart <free-variation>"]
+
+from enum import IntEnum, auto
+
+class MorphosyntacticFeature(IntEnum):
+    def __eq__(self, other):
+        return False if type(self) != type(other) else IntEnum.__eq__(self, other)
+
+class PrivativeFeature(MorphosyntacticFeature):
+    pass
+
+class BinaryFeature(MorphosyntacticFeature):
+    pass
+
+
+# Categorial Features
+
+class CategorialFeature(BinaryFeature):
+    pass
+    
+class N(CategorialFeature):
+    pos = auto()
+    neg = auto()
+    
+class V(CategorialFeature):
+    pos = auto()
+    neg = auto()
+    
+class F(CategorialFeature):
+    pos = auto()
+    neg = auto()

```

Comment:
I guess `N` stands for noun, `V` for verb but I don't know what `F` stands for. Comments are welcome here.

Reply:
Yes I will explain this.  F stands for Functional.